src/layout/LikertItem/LikertItemComponent.tsx

```diff
@@ -45,6 +47,9 @@ const RadioGroupTableRow = forwardRef<HTMLTableRowElement, IControlledRadioGroup
   const groupContainer =
     node.parent instanceof BaseLayoutNode && node.parent.isType('Likert') ? node.parent : undefined;
 
+  const item = useNodeItem(props.node) as NodeItemFromNode<BaseLayoutNode<'LikertItem'>>;
+  const columns = item.columns;
```

Comment:
You can also pick one value directly:
```suggestion
  const columns = useNodeItem(props.node, (i) => i.columns);
```

Reply:
If I do that I just get this error:
> Property 'columns' does not exist on type 'NodeItemFromNode<BaseLayoutNode<"LikertItem"> | BaseLayoutNode<"RadioButtons">>'.